site2/docs/functions-develop.md

```diff
@@ -0,0 +1,570 @@
+---
+id: functions-develop
+title: Develop Pulsar Functions
+sidebar_label: Develop functions
+---
+
+This tutorial walks you through how to develop Pulsar Functions.
+
+## Available APIs
+In Java, Python, and Go, you have two options to write Pulsar Functions.
+
+Interface | Description | Use cases
+:---------|:------------|:---------
+Language-native interface | No Pulsar-specific libraries or special dependencies required (only core libraries from Java/Python/Go). | Functions that do not require access to the function [context](#context).
+Pulsar Function SDK for Java/Python/Go | Pulsar-specific libraries that provide a range of functionality not provided by "native" interfaces. | Functions that require access to the function [context](#context).
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--Java-->
+<!--Python-->
+In Python, the language-native function, which adds an exclamation point to all incoming strings and publishes the resulting string to a topic, has no external dependencies.
```

Comment:
We can probably just mention this before the the code examples for all languages.  There is no point to mention this again and again for each language

Reply:
Sure. Good idea.